bot/cogs/config.py

```diff
@@ -218,6 +231,9 @@ async def setup(self, ctx: RoboContext, *, flags: SetupFlags) -> None:
                 default_layout=discord.ForumLayoutType.list_view,
                 available_tags=forum_tags,
             )
+            tc_webhook = await ticket_channel.create_webhook(
+                name="Rodhaj User Proxy Webhook", avatar=avatar_bytes
```

Comment:
Shouldn't this take an environment variable or something configurable for the webhook name?

Reply:
As discussed in a vc call, no. In fact, this is the same exact scheme used by PluralKit. As they dispatch webhooks, in order for proxies to latch onto those webhooks, they create a webhook for every single channel possible, and the webhook is called "PluralKit Proxy Webhook". In this case, it's the same exact thing. The user doesn't know as the actual name and details is passed in when sending a webhook message instead.